src/Uno.Foundation/UriExtensions.cs

```diff
@@ -26,6 +26,10 @@ public static class UriExtensions
 		}
 
 
+#if __ANDROID__
+		internal static Uri TrimEndUriSlash(this Uri uri) => uri.AbsolutePath.EndsWith("/") ? new Uri(uri.ToString()[..^1]) : uri;
```

Comment:
`.ToString()` and `OriginalString` behave differently, particularly when dealing with spaces or other similar escaped characters. 

I'd rather see this:
```suggestion
		internal static Uri TrimEndUriSlash(this Uri uri) => uri.OriginalString.TrimEnd('/');
```

Reply:
@jeromelaban , this will create a new instance from Uri all the time. Any concern about that? 

`internal static Uri TrimEndUriSlash(this Uri uri) => uri.OriginalString.TrimEnd('/');`

`internal static Uri TrimEndUriSlash(this Uri uri) => new Uri(uri.OriginalString.TrimEnd('/'));`

Reply:
Even if it's a low-traffic area, that's true indeed, thanks. The `OriginalString` comment is still valid, though.

Reply:
I did the recomended change. Thank you!